### PR TITLE
Mobile 3833

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: src/addons/block/starredcourses/components/starredcourses/starredcourses.ts [hunk 1]
```diff
@@ -129,54 +137,54 @@ export class AddonBlockStarredCoursesComponent extends CoreBlockBaseComponent im
...
-     * Fetch the courses.
-     *
-     * @return Promise resolved when done.
+     * @inheritdoc
      */
     protected async fetchContent(): Promise<void> {
         const showCategories = this.block.configsRecord && this.block.configsRecord.displaycategories &&
             this.block.configsRecord.displaycategories.value == '1';
 
+        // @TODO: Sort won't coincide with website because timemodified is not informed.
```
Comment (dpalou) [inline src/addons/block/starredcourses/components/starredcourses/starredcourses.ts:146]: You can probably use block_starredcourses_get_starred_courses.

File: src/core/services/sites.ts [hunk 2]
```diff
@@ -1090,32 +1090,35 @@ export class CoreSitesProvider {
     }
 
     /**
-     * Get the list of sites stored, sorted by URL and full name.
+     * Get the list of sites stored, sorted by sitename, URL and fullname.
      *
-     * @param ids IDs of the sites to get. If not defined, return all sites.
      * @return Promise resolved when the sites are retrieved.
      */
-    async getSortedSites(ids?: string[]): Promise<CoreSiteBasicInfo[]> {
```
Comment (dpalou) [inline src/core/services/sites.ts:1098]: I'd keep this parameter. We use it in getSites for certain cases, maybe in the future we want to get a sorted list in those same use cases.

**Review by dpalou — CHANGES_REQUESTED** [review]

File: src/core/services/sites.ts (hunk 2 above)
Comment (crazyserver) [reply src/core/services/sites.ts:1098]: DOne

File: src/addons/block/starredcourses/components/starredcourses/starredcourses.ts (hunk 1 above)
Comment (crazyserver) [reply src/addons/block/starredcourses/components/starredcourses/starredcourses.ts:146]: Opened MOBILE-3923 as a follow up

**Review by dpalou — APPROVED** [review]

**Pull request merged**